functions/auto-generate-perf-review.ts

```diff
@@ -64,9 +66,10 @@ export async function onRequest(
   }
 
   const smarts = new AutoPerfReviewGenerator(context.env.OPENAI_KEY);
+  const attributes = JSON.parse(params.attributes) as WorkAttribute[];
```

Comment:
Please add some extra validation about attributes.
AFAIK we might get something better than JSON encoding these things.
Via the following "trick"
`foo?bar=10&bar=20` should encoe `bar` as an array with `[10, 20]` as value.
IDK if `fetch`, or `URLSearchParams` is smart enough to behave like this.
But it's worth a try.
Otherwise add a techdebt item to migrate this to POST with a JSON body.
After that you can express some constraints of `attributes` (minLength, maxLength, etc) in the JSON schema

Reply:
https://github.com/horia141/auto-perf-review/issues/32
seems POST might be the more elegant solution. Tried w/ it as a first option, but failed.